pallets/dapps-staking/src/pallet/mod.rs

```diff
@@ -307,7 +318,12 @@ pub mod pallet {
             // a new era is triggered. This code is just a safety net to ensure nothing is broken
             // if we fail to do that.
             if PalletDisabled::<T>::get() || T::ForcePalletDisabled::get() {
-                return T::DbWeight::get().reads(1);
+                return T::DbWeight::get().reads(3);
```

Comment:
Why do we charge for 3 reads? shouldn't it be 2 here for `PalletDisabled` and `ForcePalletDisabled`?

Reply:
We have to account for the whitelisted storage, the new one and the old one, it's not account for anywhere.

In this case it's the `PalletDisabled`, `CurrentEra` and `DecommissionStarted`.
These entries are whitelisted and repeatedly used in all extrinsics, but are never charged for.